RFS/src/main/java/org/opensearch/migrations/bulkload/common/http/ConnectionContext.java

```diff
@@ -140,6 +143,9 @@ public static class TargetAdvancedArgs {
         @Parameter(names = {
             "--target-compression" }, description = "**Advanced**. Allow request compression to target", required = false)
         public boolean compressionEnabled = false;
+
+        @Parameter(names = { "--target-version-override" }, description = "**Advanced**. Allow overriding the version handling for the target cluster.", converter = VersionConverter.class)
```

Comment:
I'm not sure if this is an Advanced feature until 7.x is supported as a RemoteWriter or OS Compatibility flag is handled elsewhere

Reply:
This feature should only be used when the target cluster has `override_main_response_version` enabled and metadata / rfs cannot trust the version number returned from the cluster, seems advanced to me.